javascriptv3/example_code/bedrock-runtime/models/amazon_titan/titan_text.js

```diff
@@ -0,0 +1,76 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { fileURLToPath } from "url";
+
+import { FoundationModels } from "../../config/foundation_models.js";
+import { defaultProvider } from "@aws-sdk/credential-provider-node";
+import {
+  BedrockRuntimeClient,
+  InvokeModelCommand,
+} from "@aws-sdk/client-bedrock-runtime";
+
+/**
+ * @typedef {Object} ResponseBody
+ * @property {Object[]} results
+ */
+
+/**
+ * Invokes an Amazon Titan Text generation model.
+ *
+ * @param {string} prompt - The input text prompt for the model to complete.
+ * @param {string} [modelId] - The ID of the model to use. Defaults to "amazon.titan-text-express-v1".
+ * @returns {Promise<string[]>} The inference response from the model.
+ */
+export const invokeModel = async (prompt, modelId) => {
```

Comment:
As mentioned above, you can standardize the shape of `invokeModel` for consistency. An `Invoker` type could be created in one place (e.g. foundation_models) and reused across all of the models.